azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/configuration/AppLocalizationProvider.kt

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.configuration
+
+import android.content.Context
+
+internal class AppLocalizationProvider : LocalizationProvider {
+    private lateinit var language: String
+    private var customTranslation: Map<String, String>? = null
+
+    override fun apply(localeConfig: LocalizationConfiguration) {
+        val supportedLocales = getSupportedLocales()
+        if (supportedLocales.contains(localeConfig.languageCode)) {
+            language = localeConfig.languageCode
+        }
+        if (localeConfig.customTranslation != null) {
+            customTranslation = localeConfig.customTranslation
+        }
+    }
+
+    override fun getLocalizedString(localeKey: String, sdkLocale: String): String {
```

Comment:
Two methods with different API's but same name, needs documentation and/or better names on why to use one or the other?